test/testmathlib.cpp

```diff
@@ -1463,23 +1463,12 @@ class TestMathLib : public TestFixture {
         ASSERT_EQUALS("0.0", MathLib::toString(0.0));
         ASSERT_EQUALS("0.0", MathLib::toString(+0.0));
         ASSERT_EQUALS("0.0", MathLib::toString(-0.0));
-        // float (trailing f or F)
```

Comment:
Since we did not have a specialization for `float` and `long double` these were actually using the default implementation which is probably not what we wanted. Since we do not use those types in the code these tests could be removed and no further specializations were necessary to implement.